src/Ledger/Tally.lagda

```diff
@@ -52,27 +56,32 @@ private
            c : Coin
            addr : RwdAddr
            a : GovAction
+           prev : NeedsHash a
 \end{code}
 \begin{code}
+modifyMatch : ∀ {a} {A : Set a} → (A → Bool) → (A → A) → List A → List A
```

Comment:
I think a function of type `∀ {a} {A : Set a} → (A → Maybe A) → List A → List A` would be more convenient.